ui/v2.5/src/components/Tagger/Tagger.tsx

```diff
@@ -168,9 +167,6 @@ const TaggerList: React.FC<ITaggerListProps> = ({
       .filter((s) => s.stash_ids.length === 0)
       .map((s) => s.id);
 
-    // clear search errors
-    setSearchErrors({});
```

Comment:
I'm not sure why this was done. With this change, if the text search fails but the fingerprint search does not, then it still shows the error message and does not show the fingerprint matched scene.

Reply:
The issue was that if a fingerprint wasn't found the scene state would flip from `network error` to `no results found` despite no search having been performed. I hadn't considered your scenario though. I've corrected this now so the scene results are wiped if the query fails, and reinstated the search error clearing.